src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -70,6 +70,11 @@ public class WebResponseObject
         /// </summary>
         public string StatusDescription => WebResponseHelper.GetStatusDescription(BaseResponse);
 
+        /// <summary>
+        /// Gets or sets the output file path.
+        /// </summary>
+        public string? OutFile { get; set; }
```

Comment:
should this be an internal set?

Reply:
I agree (`public string? OutFile { get; internal set; }`)

@SteveL-MSFT, @iSazonov, just to confirm: Is it generally fine to extend public classes with new members?

Other than that, LGTM - thanks for taking this on, @jshigetomi.

Reply:
@mklement0 Removing public element is a breaking change, adding is ok.

Reply:
@jshigetomi Please address the request.

Reply:
@iSazonov Thank you for bringing this to my attention. I misread this chain and thought there were no actionable items.